core-requirements.txt

```diff
@@ -10,3 +10,4 @@ psutil>=5.6.3
 requirements-parser>=0.2.0
 shap>=0.35.0
 texttable>=1.6.2
+imbalanced-learn>=0.7.0
```

Comment:
@jeremyliweishih what is this?

Reply:
The package that I'm using to down sample and balance classes! Using selection under sampling to do so: https://imbalanced-learn.readthedocs.io/en/stable/under_sampling.html#prototype-selection